units/XNL0403/XNL0403_Script.lua

```diff
@@ -1,21 +1,19 @@
--- experimental hover unit crawler
-
-local AddLights = import('/lua/nomadsutils.lua').AddLights
-local NExperimentalHoverLandUnit = import('/lua/nomadsunits.lua').NExperimentalHoverLandUnit
 local NomadsWeaponsFile = import('/lua/nomadsweapons.lua')
-local TacticalMissileWeapon1 = import('/lua/nomadsweapons.lua').TacticalMissileWeapon1
-local MissileWeapon1 = NomadsWeaponsFile.MissileWeapon1
-local GattlingWeapon1 = NomadsWeaponsFile.GattlingWeapon1
-local StrategicMissileWeapon = import('/lua/nomadsweapons.lua').StrategicMissileWeapon
-local ParticleBlaster1 = NomadsWeaponsFile.ParticleBlaster1
-
-local explosion = import('/lua/defaultexplosions.lua')
 local EffectTemplate = import('/lua/EffectTemplates.lua')
-local GetRandomInt = import('/lua/utilities.lua').GetRandomInt
 local NomadsEffectTemplate = import('/lua/nomadseffecttemplate.lua')
-local SlowHover = import('/lua/defaultunits.lua').SlowHoverLandUnit
+local explosion = import('/lua/defaultexplosions.lua')
+local NExperimentalHoverLandUnit = import('/lua/nomadsunits.lua').NExperimentalHoverLandUnit
+local GetRandomInt = import('/lua/utilities.lua').GetRandomInt
+local SlowHoverLandUnit = import('/lua/defaultunits.lua').SlowHoverLandUnit
 
-XNL0403 = Class(NExperimentalHoverLandUnit, SlowHover) {
+local MissileWeapon1 = NomadsWeaponsFile.MissileWeapon1
+local GattlingWeapon1 = NomadsWeaponsFile.GattlingWeapon1
+local TacticalMissileWeapon1 = NomadsWeaponsFile.TacticalMissileWeapon1
+local StrategicMissileWeapon = NomadsWeaponsFile.StrategicMissileWeapon
+
+--- Experimental Hover Crawler
```

Comment:
https://github.com/FAForever/nomads/blob/80f08f97381ececa90de9f6e8d5b06946513e7ec/units/XNL0403/XNL0403_unit.bp#L2
```suggestion
--- Experimental Hover Missile Tank
```